source/includes/_feature-deriving.md

```diff
@@ -461,8 +461,8 @@ to produce a set of joint values that can be used as weights. Note that
 available weight variables are shared by all; you may not create private 
 weights. To create a weight variable, POST a JSON variable definition to the 
 variables catalog describing the properties of the weight variable, with an 
-"derivation" member indicating to use the "rake" function, which takes a 
-"targets" argument containing an array of Crunch Tables of targets:
+"derivation" member indicating to use the "rake" function, which takes arguments
```

Comment:
We use fancy quotes in some places, and straight quotes in others, should this all be the same?